Glade2d/Game.cs

```diff
@@ -90,19 +101,36 @@ public void Update()
         /// </summary>
         public void Draw()
         {
+            var accessTime = TimeSpan.Zero;
+            var drawTime = TimeSpan.Zero;
+            var startTime = _stopwatch.Elapsed;
             Renderer.Reset();
+            var resetTime = _stopwatch.Elapsed;
             var screen = GameService.Instance.CurrentScreen;
             if (screen != null)
             {
                 // TODO: this is a hack, figure out how to protect list
                 // while also making it available to the renderer
                 var sprites = screen.AccessSpritesForRenderingOnly();
+                Console.WriteLine($"Sprite count: {sprites.Count}");
+                accessTime = _stopwatch.Elapsed;
                 for (var i = 0; i < sprites.Count; i++)
                 {
                     Renderer.DrawSprite(sprites[i]);
                 }
+
+                drawTime = _stopwatch.Elapsed;
             }
+            
             Renderer.RenderToDisplay();
+            var totalTime = _stopwatch.Elapsed;
+            
+            Console.WriteLine($"Draw timings: " +
```

Comment:
Did you intend to leave an every-frame console write in? This could at least use the logger with a level so it could be suppressed?

Reply:
We discussed on slack and it makes sense to leave this in for now. It's low overhead and we're actively using it as we make perf changes.